### PR TITLE
Leadspace (Hero) #98

### PR DESCRIPTION
**Pull request by arumsey**
Fix #98

- add initial leadspace block
- add primary, secondary, tertiary buttons
- add event variant
- add image variants

Test URLs:
Before: https://main--merative2--hlxsites.hlx.page/drafts/arumsey/healthcare-analytics
After: https://issue-98-leadspace--merative2--hlxsites.hlx.page/drafts/arumsey/healthcare-analytics

Screen Shot 2023-02-03 at 5 02 25 PM

### CODE REVIEW

**Review by amol-anand — COMMENTED** [review]
Please fix linting issues. some other comments inline

File: blocks/carousel/carousel.css [hunk 1]
```diff
@@ -29,16 +36,6 @@
     transition: all 0.5s;
 }
 
-.carousel .carousel-slide > div:nth-child(1) {
```
Comment (amol-anand) [inline blocks/carousel/carousel.css:32]: I see you've removed some stuff from carousel.css is that for Leadspace block specifically or just some other fixes to carousel that are just in the same PR?

**Review by amol-anand — COMMENTED** [review]
Couple of updates:
1. I fixed the linting issues for now.
2. Open question on carousel changes.
3. I like the authoring experience for the Leadspace. Once you're done with whatever you need to do, let me know and I can merge it in.

File: blocks/cta/cta.css [hunk 2]
```diff
@@ -65,6 +65,8 @@
     font-weight: 500;
     letter-spacing: var(--letter-spacing-1);
     line-height: var(--line-height-160);
+    border: none;
```
Comment (arumsey) [inline blocks/cta/cta.css:68]: Undo default button styling

File: blocks/leadspace/leadspace.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+  const controlBtn = createTag('div', { class: `video-control ${type}` });
+  controlBtn.innerHTML = '<button type="button"><span aria-hidden="true"></button>';
+  const btn = controlBtn.querySelector('button');
+  btn.setAttribute('aria-label', `${type} video`);
+  btn.style.visibility = visible ? 'visible' : 'hidden';
+  return controlBtn;
+};
+
+const openVideoOverlay = () => {
+  alert('I was clicked!');
```
Comment (arumsey) [inline blocks/leadspace/leadspace.js:37]: Will be implemented once video overlay support is added.

**Review by amol-anand — APPROVED** [review]

**Pull request merged**